src/hit/api/linearalgebra/encryptedmatrix.cpp

```diff
@@ -22,8 +22,10 @@ namespace hit {
                                           const protobuf::EncryptedMatrix &encrypted_matrix) {
         height_ = encrypted_matrix.height();
         width_ = encrypted_matrix.width();
+        if (height_ == 0 && width_ == 0) {
+            return;
+        }
```

Comment:
Nit: may add comments (in case we forget) here to indicate this means the matrix is not initialized, and then no validation will be made. Same comments for other two changes.